src/app/common/stores/applications/SsvWeb/Wallet.store.ts

```diff
@@ -15,15 +15,24 @@ import SsvStore from '~app/common/stores/applications/SsvWeb/SSV.store';
 import OperatorStore from '~app/common/stores/applications/SsvWeb/Operator.store';
 import MyAccountStore from '~app/common/stores/applications/SsvWeb/MyAccount.store';
 import NotificationsStore from '~app/common/stores/applications/SsvWeb/Notifications.store';
-import { changeCurrentNetwork, getCurrentNetwork, NetworkDataType, NETWORKS, NETWORKS_DATA } from '~lib/utils/envHelper';
+import {
+  changeCurrentNetwork,
+  getCurrentNetwork,
+  GOERLI_NETWORK_ID,
+  HOLESKY_NETWORK_ID,
+  inNetworks,
+  NetworkDataType,
+  NETWORKS,
+  NETWORKS_DATA,
+  toHexString,
+} from '~lib/utils/envHelper';
 
 const WALLET_CONNECTED = 'WalletConnected';
 
-const GOERLI_NETWORK_ID = 5;
-
 const TOKEN_NAMES = {
   [NETWORKS.MAINNET]: 'ETH',
   [NETWORKS.GOERLI]: 'GoerliETH',
+  [NETWORKS.HOLESKY]: 'ETH',
```

Comment:
@meshin-dev 
It's just ETH? Not something like HoleskyETH?

Reply:
Yeah I don't see the reason to name it differently.
According to guides it should be ETH.
Just in testent not mainnet.

Example from others: https://metaschool.so/rpc/holesky